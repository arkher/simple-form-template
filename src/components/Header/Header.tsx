const Header = () => {
  return (
    <div className="flex flex-col items-start pb-4" data-testid="header">
      <h1 className="text-4xl text-black">Valide sua senha</h1>
    </div>
  );
};

export default Header;
