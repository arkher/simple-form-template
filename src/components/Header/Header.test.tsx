import React from "react";
import { render, screen } from "@testing-library/react";
import Header from "./Header";

describe("Header", () => {
  it("should render", () => {
    render(<Header />);
    expect(screen.getByText("Valide sua senha")).toBeTruthy();
  });

  it("should match snapshot", () => {
    render(<Header />);
    expect(screen.getByTestId("header")).toMatchSnapshot();
  });
});
