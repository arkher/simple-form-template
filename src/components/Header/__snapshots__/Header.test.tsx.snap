// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<div
  class="flex flex-col items-start pb-4"
  data-testid="header"
>
  <h1
    class="text-4xl text-black"
  >
    Valide sua senha
  </h1>
</div>
`;
